using_images/db_images/mongodb.adoc

```diff
@@ -10,50 +10,24 @@
 toc::[]
 
 == Overview
-The OpenShift https://github.com/openshift/mongodb/tree/master[MongoDB] image
-repository contains the sources and Dockerfiles for building various versions of
-MongoDB images. The resulting images can be run by Docker.
+OpenShift provides a Docker image for running MongoDB.  This image can provide database services based on username, password, and database name settings provided via configuration.
 
 == Versions
 Currently, OpenShift only provides version https://github.com/openshift/mongodb/tree/master/2.4[2.4] of MongoDB.
 
-== Base Images
+== Images
 
 The following base images are currently available:
```

Comment:
It would be good to rephrase that to something along these lines:
"MongoDB image comes in two flavours, depending on your needs:"